### PR TITLE
hopefully fix Windows git hangs with RemoteClassLoader.force

### LINKED ISSUE #203: regression: Windows jobs consistently freezing after git clean, before build script
e.g. https://scala-ci.typesafe.com/job/scala-2.13.x-integrate-windows/29/consoleFull

I had hoped this was just an intermittent thing, but for several days now all Windows jobs are affected.

I don't think this is the same as https://github.com/scala/scala-jenkins-infra/issues/143, but who knows. I did check that longpaths is set to true in user jenkins' global gitconfig

@adriaanm any ideas? I'm stumped. we won't able to release 2.12.1 without finding some solution or workaround, as the -package-  jobs hang too, see e.g. the string of failures at https://scala-ci.typesafe.com/view/scala-2.11.x/job/scala-2.12.x-release-package-windows/

### PR DESCRIPTION
**Pull request by SethTisue**
hopefully fixes #203, see remarks there for details

### CODE REVIEW

**Comment by SethTisue** [comment]
blocked for now by #231

**Comment by SethTisue** [comment]
hmm, I don't think I put this in the right place... uploading it and running `chef-client` doesn't make the setting show up at https://scala-ci.typesafe.com/computer/jenkins-worker-windows-publish/configure

File: attributes/worker.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@
     jvmBin                           = "/cygdrive/c#{jvmHome}/bin"
   
     # deal with weirdness in java using registry keys to find the homedir (c:\users\jenkins), instead of the cygin home (/home/jenkins or y:\jenkins)
-    jvmDirOptions = "-Duser.home=#{jenkinsHome.gsub(/\\/,'/')} -Djava.io.tmpdir=#{jenkinsTmp}" # jenkins doesn't quote properly
+    jvmDirOptions = "-Duser.home=#{jenkinsHome.gsub(/\\/,'/')} -Djava.io.tmpdir=#{jenkinsTmp} -Dhudson.remoting.RemoteClassLoader.force=com.sun.jna.Native" # jenkins doesn't quote properly
```
Comment (retronym) [inline attributes/worker.rb:22]: If we can't find the right spot for this in the recipe, we could resort to putting it in the global `_JAVA_OPTIONS` on the worker.

Comment (SethTisue) [reply attributes/worker.rb:22]: Adriaan thinks he got the change to go through. running chef-client on jenkins-master wasn't enough

**Pull request merged**